app/src/main/java/net/gini/pay/app/review/ReviewActivity.kt

```diff
@@ -21,6 +22,7 @@ class ReviewActivity : AppCompatActivity() {
     private val viewModel: ReviewViewModel by viewModel()
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        viewModel.giniBusiness.setSavedStateRegistryOwner(this, viewModel.viewModelScope)
```

Comment:
Can you please add documentation for the `setSavedStateRegistryOwner` method? If I'm not mistaken clients will need to use it in order to enable state restoration.

Reply:
I added